common/openapi.yml

```diff
@@ -749,10 +765,14 @@ components:
       description: |
         json object containing the admin provided information to update a pipeline with
       type: object
-      required: [ workspaceId ]
+      required: [ workspaceProject, workspaceName ]
```

Comment:
for now, only workspace project and name are required. workspace id might become required in the future, but currently GCP workspaces don't have user-facing workspace ids.

Reply:
that's a good point. probably none should be required here, since the intent is to allow updating any of them independently.

Reply:
instead, we agreed to remove workspaceId from these endpoints, so these two fields should be required after all (here and in the update function)